src/pyuvdata/uvdata/uvdata.py

```diff
@@ -1397,7 +1397,9 @@ def _consolidate_phase_center_catalogs(
             # match, so this is functionally the same source
             self._update_phase_center_id(match_id, new_id=cat_id)
             # look_in_catalog ignores the "info_source" field, so update it to match
-            self.phase_center_catalog[cat_id]["info_source"] = cat_entry["info_source"]
+            info_source = cat_entry.get("info_source")
+            if info_source is not None:
+                self.phase_center_catalog[cat_id]["info_source"] = info_source
```

Comment:
Can you change this to:
`self.phase_center_catalog[cat_id]["info_source"] = cat_entry.get("info_source")`
Realizing this is pretty minor, although the plan for the phase dict is to become an actual object, where these will become optional attributes (which always exist _as_ attributes on the object, but are filled in w/ None when not set), and this will basically make the code easier to migrate.